src/renderer/actions/databases.js

```diff
@@ -60,6 +64,35 @@ export function saveDatabaseDiagram(diagramJSON) {
   };
 }
 
+export function exportDatabaseDiagram(diagram, imageType) {
+  return async (dispatch) => {
+    dispatch({ type: EXPORT_DIAGRAM_REQUEST });
+    try {
+      // fix - reapply css roles which html2canvas ignores for some reason
+      $('.link-tools, .marker-arrowheads', diagram).css({ display: 'none' });
+      $('.link, .connection', diagram).css({ fill: 'none' });
```

Comment:
You're absolutely right! I was bit tired when writing this fix, so I forgot to move it. Good catch :)